OmniKit/PumpManager/PodCommsSession.swift

```diff
@@ -457,6 +457,11 @@ public class PodCommsSession {
     public func cancelDelivery(deliveryType: CancelDeliveryCommand.DeliveryType, beepType: BeepType) -> CancelDeliveryResult {
         var message: [MessageBlock]
 
+        // Pod will fault or not respond if its setup is not complete, just return a noResponse error if command can't be run.
+        guard podState.setupProgress == .completed else {
+            return CancelDeliveryResult.certainFailure(error: PodCommsError.noResponse)
```

Comment:
Returning a noResponse here for something we didn't attempt could create some confusion. Would rather you just remove this check, since the more sensical code path is to just not attempt in this state, which you've done below.

Reply:
Removed extra check in cancelDelivery itself to prevent a potential pod fault if cancelDelivery() is called incorrectly as requested